### ISSUE TITLE
[semantic-issue] Add semantics for sklearn RandomForestClassifier

### ISSUE THREAD

**Issue by osryde** [spangea/Pyra#170]
**Description**
Invokes the constructor to instantiate a RandomForestClassifier object.
Returns a RandomForestClassifier object.

**Minimal running example**
```
from sklearn.ensemble import RandomForestClassifier

rfc = RandomForestClassifier()
```

**Links to documentation**
https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.RandomForestClassifier.html

**Note (optional)**

Notebook: 0-91-accuracy-score-for-4-models-lr-rc-gbc-rfc.py
